tst/EnergyPlus/unit/DXCoils.unit.cc

```diff
@@ -279,3 +279,525 @@ namespace EnergyPlus {
 	}
 
 }
+
+TEST( DXCoilsTest, TestMultiSpeedDefrostCOP )
```

Comment:
@nmerket You need to move this and the other unit test within the ```namespace EnergyPlus { }``` in this unit test file. Also, if you switch this to ```TEST_F( HVACFixture, DXCoils_TestMultiSpeedDefrostCOP )``` the compile errors should go away.